admin.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require_once './database.php';
+
+//get projects
+$sql = "SELECT * FROM `project`;";
+$query = $db->prepare($sql);
+$query->execute();
+$projects = $query->fetchAll();
```

Comment:
should turn this into a function like `getAllProjects`, you could then reuse it on the front-end

Reply:
will refactor this post-MVP